src/pint/models/timing_model.py

```diff
@@ -895,8 +905,8 @@ def remove_component(self, component):
         cp, co_order, host, cp_type = self.map_component(component)
         host.remove(cp)
 
-    def _locate_param_host(self, components, param):
-        """Search for the parameter host component.
+    def _locate_param_host(self, param):
+        """Search for the parameter host component in the timing model.
```

Comment:
Please clarify this. Perhaps "Find the component containing the given parameter." Does this expect the string parameter name or an actual parameter object?

Reply:
Does this accept aliases?